server/src/main/java/com/angelomelonas/grpcwebchat/grpc/ChatService.java

```diff
@@ -1,25 +1,39 @@
-package com.angelomelonas.grpcwebchat;
+package com.angelomelonas.grpcwebchat.grpc;
 
 import com.angelomelonas.grpcwebchat.Chat.Message;
 import com.angelomelonas.grpcwebchat.Chat.MessageRequest;
 import com.angelomelonas.grpcwebchat.Chat.SubscriptionRequest;
+import com.angelomelonas.grpcwebchat.ChatServiceGrpc.ChatServiceImplBase;
+import com.angelomelonas.grpcwebchat.chat.ChatSession;
+import com.angelomelonas.grpcwebchat.subscription.SubscriptionService;
 import io.grpc.stub.StreamObserver;
 import org.lognet.springboot.grpc.GRpcService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.UUID;
+
 @GRpcService
-public class ChatService extends ChatServiceGrpc.ChatServiceImplBase {
+public class ChatService extends ChatServiceImplBase {
+    private SubscriptionService subscriptionService;
+    private ChatSession chatSession;
```

Comment:
maybe just go simple and store the sessions in a list or map ... thereafter as things get more clearer you will create more suitable classes